flowkit/config/json/deploy_test.go

```diff
@@ -151,3 +151,33 @@ func Test_DeploymentAdvanced(t *testing.T) {
 	assert.Equal(t, "KittyItemsMarket", alice.Contracts[1].Name)
 	assert.Len(t, alice.Contracts[1].Args, 0)
 }
+
+func Test_EmptyEmulatorDeployment(t *testing.T) {
+	b := []byte(`{
+		"emulator": {}
+	}`)
+
+	result := []byte(`{
+		"emulator": {"":[]}
```

Comment:
Seems odd to replace it with this. Is there no way to keep it as it was?

Reply:
I added a else if to check if only network is set in a deployment and allow for a empty deployments object